openshift/templates/sidecar/deploy.yaml

```diff
@@ -144,6 +144,12 @@ objects:
             - name: wordpress-volume
               persistentVolumeClaim:
                 claimName: ${APP_NAME}-wordpress--${POOL_NAME}
+            - name: ${APP_NAME}-backup-file
```

Comment:
Minor, but "file" doesn't seem the most descriptive. I'd guess you're just trying to differentiate it from the db?

Reply:
this is only used internally, within the deploy, and doesn't get captured outside of the deploy file.  Appreciate the feedback.